apps/cowswap-frontend/src/legacy/state/orders/updaters/SpotPricesUpdater.ts

```diff
@@ -55,18 +55,25 @@ function useMarkets(chainId: SupportedChainId): MarketRecord {
 }
 
 interface UseUpdatePendingProps {
+  isWindowVisibleRef: React.MutableRefObject<boolean>
```

Comment:
Why use a ref for this rather than a boolean?

Reply:
To avoid excessive calls. If we use just a boolean then we have to add it to useEffect() deps and it will trigger a call on window focus